src/modules/Builder/pages/BuilderApplet/BuilderApplet.utils.tsx

```diff
@@ -135,18 +136,36 @@ export const getNewPerformanceTask = ({
 
   const propsByTypeObj = {
     [PerformanceTasks.Flanker]: defaultFlankerProps,
-    [PerformanceTasks.Gyroscope]: defaultGyroscopeAndTouchProps,
-    [PerformanceTasks.Touch]: defaultGyroscopeAndTouchProps,
+    [PerformanceTasks.Gyroscope]: {
+      responseType: ItemResponseType.Gyroscope,
+      ...defaultGyroscopeAndTouchProps,
+    },
+    [PerformanceTasks.Touch]: {
+      responseType: ItemResponseType.Touch,
+      defaultGyroscopeAndTouchProps,
+    },
   };
 
   const defaultPropsByType = type
-    ? propsByTypeObj[type as unknown as EditablePerformanceTasks] || {}
-    : {};
+    ? propsByTypeObj[type as unknown as EditablePerformanceTasks] || { responseType: '' }
+    : { responseType: '' };
+
+  const { responseType, ...config } = defaultPropsByType;
 
   return {
     name,
     description,
-    ...defaultPropsByType,
+    items: [
+      {
+        id: undefined,
+        key: uuidv4(),
+        name: `${responseType}`,
+        responseType,
+        question: '',
```

Comment:
I think question and responseValues can be removed from here, they are not mandatory